lib/reports/cost_report.rb

```diff
@@ -9,6 +9,8 @@ class CostReport
     attr_accessor :organization, :logger, :maxlines, :target_cost
 
     def initialize(org = nil, cost: Settings.target_cost, lines: 1_000_000, logger: Services.logger)
+      cost ||= Settings.target_cost
```

Comment:
I'm not sure why I needed this. Maybe something is passing in `cost: nil` and this is to try to get around that? Seems like it would be worth fixing whatever was sending `cost: nil`.